pulser-core/pulser/sequence/sequence.py

```diff
@@ -370,6 +398,7 @@ def config_slm_mask(self, qubits: Iterable[QubitId]) -> None:
         # If checks have passed, set the SLM mask targets
         self._slm_mask_targets = targets
 
+    @seq_decorators.screen
```

Comment:
@HGSilveri What is this decorator for exactly?

Reply:
It blocks call to the method when the sequence is parametrized. For now, at least, this method won't work when the sequence is parametrized, and since it's just a convenience method anyway, it's simpler to just block it for now. If there is the need for it, we can extend it later on.